packages/react-utils/README.md

```diff
@@ -0,0 +1,11 @@
+# @chakra-ui/react-utils
+
+A Quick description of the component
```

Comment:
Not sure we should even include a `README.md` for this unless we plan on providing more of a description of what the purpose is

Reply:
Good point. Will make changes for the README.md.

A simple one line description for the component is sufficient.

It's helpful for contributors who may want to add more react utilities.

Reply:
Included a better description in the README of `@chakra-ui/utils`